fetch.bs

```diff
@@ -4283,32 +4307,50 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
       <!-- cache hit -->
       <ol>
-       <li><p>If <var>storedResponse</var> requires validation (i.e., it is not fresh), then set the
-       <var>revalidatingFlag</var>.
-
-       <li>
-        <p>If the <var>revalidatingFlag</var> is set and <var>httpRequest</var>'s
-        <a for=request>cache mode</a> is neither "<code>force-cache</code>" nor
-        "<code>only-if-cached</code>", then:
+       <li><p>If <a for=request>cache mode</a> is "<code>default</code>" and
+        <var>storedResponse</var> is a <a>stale-while-revalidate response</a>, then:
+        <ol>
+         <li><p>Set <var>response</var> to <var>storedResponse</var>.
+         <li><p>Set <var>response</var>'s <a for=response>cache state</a> to "<code>local</code>".
+         <li><p>In parallel perform <a for=main>main fetch</a> using <var>request</var> with
+          <ul class=brief>
+          <li><p><a for=request>cache mode</a> set to "<code>no-cache</code>".
+          <li><p><a for=request>prevent no-cache cache-control header modification flag</a> set.
+          <li><p><a for=request>service-workers mode</a> set to "<code>none</code>".
+         </ul>
```

Comment:
You're modifying the request here. It seems to me you'd need to copy it first?

Reply:
done